client/state/data-layer/wpcom/sites/automated-transfer/status/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	EVERY_FIVE_SECONDS,
+	add as addRunner,
+	remove as removeRunner,
+} from 'lib/interval/runner';
+import wpcom from 'lib/wp';
+
+import {
+	AUTOMATED_TRANSFER_STATUS_REQUEST as REQUEST,
+	AUTOMATED_TRANSFER_STATUS_UPDATE as UPDATE,
+} from 'state/action-types';
+
+import { requestStatus, updateStatus } from 'state/automated-transfer/actions';
+import { transferStates } from 'state/automated-transfer/constants';
+
+const watchList = new Map();
+
+const statusMapping = {
+	complete: transferStates.COMPLETE,
+	uploading: transferStates.START,
+};
```

Comment:
Is this needed because of how I've initially named the AT statuses in the progress bar?
It'd take me no time to change it to abide to the actual naming, if that means dropping unnecessary mappings.